scan.go

```diff
@@ -238,6 +239,16 @@ func scan(pkgPatterns []string) ([]*unit.SourceUnit, error) {
 		// compare results in tests more easily.
 		pkg.Root = ""
 
+		if strings.HasPrefix(pkg.ImportPath, "_") {
+			wd, err := os.Getwd()
+			if err != nil {
+				return nil, err
+			}
+			pkg.ImportPath = repo + pkg.ImportPath[1+len(wd):]
```

Comment:
I think the best approach is just to go with your original change. Could you add a docstring at the top of the `scan()` declaration documenting this new behavior, since at first glance, it's not obvious that it would do this?